src/main/resources/data/htm/lang/de_de.json

```diff
@@ -0,0 +1,34 @@
+{
+  "text.htm.error.unlockable": "§cDieser Block kann nicht gesperrt werden",
+  "text.htm.error.no_lock": "§cDer Behälter ist nicht gesperrt",
+  "text.htm.error.not_owner": "§cDieser Behälter kann nicht gesperrt werden",
+  "text.htm.error.already_locked": "§cDieser Behälter ist bereits gesperrt",
+  "text.htm.error.lock_type": "§cUngültiger Sperrtyp",
+  "text.htm.error.flag_type": "§cUngültiger flag-Typ",
+  "text.htm.error.trust_self": "§cKann nicht sich selbst vertrauen",
+  "text.htm.error.already_trusted": "§r§a%s §cwurde bereits vertraut",
+  "text.htm.error.not_trusted": "§r§a%s §cwurde nicht vertraut",
+
+
+  "text.htm.select": "§bLinksklicke einen Block, um ihn auszuwählen",
+  "text.htm.locked": "§cGesperrt",
+  "text.htm.divider": "§b-------------------------",
+  "text.htm.type": "§b§lTyp: §r§a%s",
+  "text.htm.owner": "§b§lBesitzer: §r§a%s",
+  "text.htm.trusted": "§b§lVertraut: §r§a%s",
+  "text.htm.trusted.global": "§b§lGlobal vertraut: §r§a%s",
+  "text.htm.trust": "§a%s §bwird vertraut",
+  "text.htm.untrust": "§a%s §b wird nicht vertraut",
+  "text.htm.global": " §a§lGlobal",
+  "text.htm.transfer": "§bBehälter wurde nach §a%s transferiert",
```

Comment:
Englisch ist "$bTransferred container to §a%s" und bezieht sich auf eine Person, dem es gegeben wird. Vielleicht "Wurde %s gegeben" oder so?